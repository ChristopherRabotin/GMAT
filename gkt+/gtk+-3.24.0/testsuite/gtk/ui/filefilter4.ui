<!-- test invalid file filter markup attribute -->
<interface>
  <object class="GtkFileFilter">
    <mime-types>
      <mime-type food="good">text/plain</mime-type>
    </mime-types>
  </object>
</interface>
