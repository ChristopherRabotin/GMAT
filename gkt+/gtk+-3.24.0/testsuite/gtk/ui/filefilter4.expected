ERROR: g-markup-error-quark 4
.:5:31 attribute 'food' invalid for element 'mime-type'
